### DISCUSSION TITLE
Moving data to a different folder

### DISCUSSION THREAD

**Discussion by jqmiller-upenn** [thilojaeggi/WinGetty#66] [Q&A]
Hi everyone
I have a local install, and so far it is perfect.  I do have one quick question - is there a way to have the data folders in a specific folder structure using the docker-compose.yaml file? Right now, the data is in a folder that might get lost/recreated.

**Comment by thilojaeggi** [comment]
The folder gets placed in a docker volume so that shouldn't. happen?

**Comment by jqmiller-upenn** [comment]
I ran a docker compose pull && docker compose up -d and all my packages were lost.  I figured it overwrote everything with the docker pull.  I know in other docker containers that I have I can specifiy in the yaml file where the data actually lives and "map" that to the internal structure.
